db/db_impl/db_impl_write.cc

```diff
@@ -1821,9 +1822,11 @@ Status DBImpl::DelayWrite(uint64_t num_bytes,
   // proceed
   Status s;
   if (write_controller_.IsStopped()) {
-    // If writes are still stopped, it means we bailed due to a background
-    // error
-    s = Status::Incomplete(error_handler_.GetBGError().ToString());
+    if (!shutting_down_.load(std::memory_order_relaxed)) {
```

Comment:
Would it make more sense to return Status::Incomplete since the write was still in stopped state?

Reply:
what about return Status::ShutdownInProgress ?

Reply:
Yeah, that's better.